.github/workflows/support-window.yml

```diff
@@ -35,7 +35,7 @@ jobs:
               | from_entries
             ' > docs/support-window.json
       - name: Make SVG diagram
-        run: node --experimental-json-modules scripts/support-window > docs/support-window.svg
+        run: node --require source-map-support/register --experimental-json-modules scripts/support-window > docs/support-window.svg
```

Comment:
I'd rather not source-map support-window, and instead convert it to .js -- there aren't even any type annotations in it that I can see.

This also applies to the other .ts files in scripts/; fix-tslint.ts and generate-tsconfigs.ts each only have *one* non-`any` type annotation, so they should be .js files with a single JSDoc comment.

Reply:
- [x] I've opened https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61102#issue-1294622291 to port `scripts/*.ts` -> `scripts/*.js`.